.github/workflows/release-docker-base.yml

```diff
@@ -5,27 +5,20 @@ on:
     tags:
       - docker-base-v[0-9]+.[0-9]+.[0-9]+ # Triggered by git tags like: docker-base-v0.2.12
 
-env:
-  REPO_OWNER: "clearmatics"
-
 jobs:
 
   docker-build:
-    name: "Build docker zeth-base"
+    name: "Docker build"
     runs-on: ubuntu-20.04
     timeout-minutes: 360
 
     steps:
       - name: "Checkout"
         uses: actions/checkout@v2
 
-      - name: "Get vars from git"
-        id: git_vars
-        run: echo ::set-output name=TAG::${GITHUB_REF/refs\/tags\/docker-base-/}
-
       - name: "Create swap from file"
         run: |
-            sudo fallocate -l 21G /swapfile2
+            sudo fallocate -l 19G /swapfile2
```

Comment:
Out of curiosity, how the range `19G` chosen here?

Reply:
Not enough space on local disk for images if allocate 21G for swap